Adding tag RuntimeIdentifiers to prevent build error in xdt project

eddynaka opened the pull request:
Fix Issue # .

## Changes
(Please provide a brief description of the changes here.)

### Checklist
- [ ] I ran Unit Tests locally.
- [ ] CHANGELOG.md updated with one line description of the fix, and a link to the original issue if available.

For significant contributions please make sure you have completed the following items:

- [ ] Design discussion issue #
- [ ] Changes in public surface reviewed

The PR will trigger build, unit tests, and functional tests automatically. Please follow [these](https://github.com/Microsoft/ApplicationInsights-dotnet/blob/develop/.github/CONTRIBUTING.md) instructions to build and test locally.

### Notes for authors:
- FxCop and other analyzers will fail the build. To see these errors yourself, compile localy using the Release configuration.

### Notes for reviewers:

- We support [comment build triggers](https://docs.microsoft.com/azure/devops/pipelines/repos/github?view=azure-devops&tabs=yaml#comment-triggers)
  - `/AzurePipelines run` will queue all builds
  - `/AzurePipelines run <pipeline-name>` will queue a specific build

TimothyMothra commented:
Hey eddy, this project has some other weirdness.

- This is a PackageReference csproj, but it has a packages.config. This needs to be removed.
- The references to StyleCop should be removed. We don't need to run analyzers on test projects.
- The `<Reference Include="Microsoft.Web.XmlTransform">` needs to be replaced with `<PackageReference Include="Microsoft.Web.Xdt" Version="2.1.1" />`

can you please clean this up and check if any other props should be cleaned up?
The `RuntimeIdentifiers` might not be needed anymore because you've cleaned up so much of our errant properties. :)

eddynaka commented:
> Hey eddy, this project has some other weirdness.
> 
> * This is a PackageReference csproj, but it has a packages.config. This needs to be removed.
> * The references to StyleCop should be removed. We don't need to run analyzers on test projects.
> * The `<Reference Include="Microsoft.Web.XmlTransform">` needs to be replaced with `<PackageReference Include="Microsoft.Web.Xdt" Version="2.1.1" />`
> 
> can you please clean this up and check if any other props should be cleaned up?
> The `RuntimeIdentifiers` might not be needed anymore because you've cleaned up so much of our errant properties. :)

1. removed the packages.config
2. removed stylecop and searched in the folder for the same structure. Didn't find any. So, it looks right
3. updated the reference
4. removed the `runtimeIdentifiers`
